src/app/habits/page.tsx

```
@@ -0,0 +1,54 @@
+"use client";
+
+import React, { useState, useMemo, useEffect } from "react";
```

ESLint

[error] 'useEffect' is defined but never used.

(@typescript-eslint/no-unused-vars)